patches.json

```diff
@@ -60,6 +60,15 @@
       "2.2.2 - 2.2.3": "MAGECLOUD-1607__overhaul_cron_implementation__2.2.2.patch",
       "2.2.4": "MAGECLOUD-1607__overhaul_cron_implementation__2.2.4.patch"
     },
+    "Fix cron deadlocks and improve cron locking": {
+      "2.2.5 - 2.2.8": "MAGECLOUD-4530__fix_cron_deadlocks_and_improve_job_locking__2.2.5.patch",
+      "2.2.9": "MAGECLOUD-4530__fix_cron_deadlocks_and_improve_job_locking__2.2.9.patch",
+      ">=2.2.10 <2.3.0": "MAGECLOUD-4530__fix_cron_deadlocks_and_improve_job_locking__2.2.10.patch",
+      "2.3.0": "MAGECLOUD-4530__fix_cron_deadlocks_and_improve_job_locking__2.3.0.patch",
+      "2.3.1": "MAGECLOUD-4530__fix_cron_deadlocks_and_improve_job_locking__2.3.1.patch",
+      "2.3.2": "MAGECLOUD-4530__fix_cron_deadlocks_and_improve_job_locking__2.3.2.patch",
```

Comment:
2.3.2 has patch version if this patch should be applied to the patch version you need to change constraints

Reply:
@oshmyheliuk thanks, fixed.